src/LDLFactorizations.jl

```diff
@@ -4,22 +4,47 @@ export ldl, \
 
 using AMD, SparseArrays
 
+function col_symb(n,Ap,Ai,Cp,w,Pinv)
+  fill!(w,0)
+  @inbounds for j = 1 : n    
+    @inbounds for p = Ap[j] : (Ap[j+1]-1)
+      i = Ai[p]
+      i >= j && break #Only upper part
```

Comment:
Two blanks before `#` and one blank after it. Use all lowercase if it's just one sentence.